mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala

```diff
@@ -259,6 +259,9 @@ object MetalsInteractive:
               indexed.ctx.source,
             )
           then List((head.symbol, head.typeOpt))
+          // https://github.com/lampepfl/dotty/issues/15937
```

Comment:
Can you elaborate on which case applies this pattern?

Reply:
Added a comment!